src/test/java/core/basesyntax/service/FileReaderServiceTest.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.service.impl.FileReaderServiceImpl;
+import java.io.File;
+import java.util.List;
+import org.junit.Test;
+
+public class FileReaderServiceTest {
+    private static final String testTransactionFilePath = "src/main/resources/transaction.csv";
+
+    @Test
+    public void fileToRead_isSourceFileExist_OK() {
+        File sourceFile = new File(testTransactionFilePath);
+        assertTrue(sourceFile.exists());
+    }
+
+    @Test
+    public void listString_isNotEmpty_OK() {
+        FileReaderService fileReaderService = new FileReaderServiceImpl();
```

Comment:
you should make this variable global and initialize in method with @BeforeClass annotation, the same for other similar cases in other test classes

Reply:
Yes, sir!